src/main/java/com/walking/intensive/chapter1/task1/Task1.java

```diff
@@ -14,15 +14,24 @@
  */
 public class Task1 {
     public static void main(String[] args) {
-//        Для собственных проверок можете делать любые изменения в этом методе
-        int age = 0;
+
+        int age = 111;
 
         System.out.println(getAgeString(age));
     }
 
     static String getAgeString(int age) {
-//        Место для вашего кода
 
-        return null; // Заглушка. При реализации - удалить
+        if (age < 0 ) {
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter1/task1/Task1.java:[25](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/11986408247/job/33428365973?pr=311#step:4:26):21: ')' is preceded with whitespace. [ParenPad]